pkg/metrics/serve.go

```diff
@@ -16,19 +16,24 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/containerd/containerd/log"
+	"github.com/containerd/nydus-snapshotter/pkg/daemon/types"
 	"github.com/containerd/nydus-snapshotter/pkg/manager"
 	"github.com/containerd/nydus-snapshotter/pkg/metrics/collector"
 	"github.com/containerd/nydus-snapshotter/pkg/metrics/exporter"
 )
 
+// Default period to determine a hang IO.
+const defaultHangIOPeriod = 10 * time.Second
```

Comment:
I think `interval` is better than `period` to express

Reply:
Yes, I have replaced `period` with `interval`.